### ISSUE TITLE
test fails on windows due to platform dependent rounding

### ISSUE THREAD

**Issue by spoeschel** [skynav/ttt#165]
When I compile the ttt under Windows (7 SP1 64bit), the ttpe test case 069 fails, while all the other tests (of ttpe as well as the other tools) are passed. I get the following message:

```
-------------------------------------------------------------------------------
Test set: com.skynav.ttpe.app.PresenterTestCases
-------------------------------------------------------------------------------
Tests run: 89, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 89.938 sec <<< FAILURE! - in com.skynav.ttpe.app.PresenterTestCases
test069BidiReorderWithIsolates(com.skynav.ttpe.app.PresenterTestCases)  Time elapsed: 0.623 sec  <<< FAILURE!
java.lang.AssertionError: Expected attribute value '15.8062' but was '15.8063' - comparing <text x="15.8062"...> at /svg[1]/g[2]/svg[1]/g[1]/g[3]/g[1]/text[2]/@x to <text x="15.8063"...> at /svg[1]/g[2]/svg[1]/g[1]/g[3]/g[1]/text[2]/@x
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.assertTrue(Assert.java:41)
    at org.junit.Assert.assertFalse(Assert.java:64)
    at com.skynav.ttpe.app.PresenterTestCases.checkDifferencesXML(PresenterTestCases.java:761)
    at com.skynav.ttpe.app.PresenterTestCases.checkDifferences(PresenterTestCases.java:748)
    at com.skynav.ttpe.app.PresenterTestCases.checkDifferences(PresenterTestCases.java:707)
    at com.skynav.ttpe.app.PresenterTestCases.checkDifferences(PresenterTestCases.java:684)
    at com.skynav.ttpe.app.PresenterTestCases.checkDifferences(PresenterTestCases.java:664)
    at com.skynav.ttpe.app.PresenterTestCases.maybeCheckDifferences(PresenterTestCases.java:643)
    at com.skynav.ttpe.app.PresenterTestCases.performPresentationTest(PresenterTestCases.java:540)
    at com.skynav.ttpe.app.PresenterTestCases.performPresentationTest(PresenterTestCases.java:504)
    at com.skynav.ttpe.app.PresenterTestCases.test069BidiReorderWithIsolates(PresenterTestCases.java:400)
```

It seems to be just a kind of rounding error, therefore for testing I replaced the four occurences of `15.8062` within the ttps000001.svg of test-069-bidi-reorder-with-isolates.expected.zip by `15.8063`. This fixed it, though this is of course only a workarond.

Edit: the problem also occurs on Linux (Ubuntu 14.04 AMD64).

**Comment by skynavga** [comment]
This issue also shows up on MacOSx when using JDK 1.8 (but not JDK 1.7), so it appears it is JDK specific rather than platform specific.

**Comment by skynavga** [comment]
I have disabled this failing test (test069BidiReorderWithIsolates) until I can rebase the expectation file and move to JDK 1.8 completely.

**Comment by spoeschel** [comment]
I just forgot to post the used Java version in the first place; it is JDK 1.8 as well.
Quite strange error...

**Comment by skynavga** [comment]
Fixed in 9d22758d2154403fdf47bb1e575c78b450df5988 by regenerating reference output (zip) using JDK 1.8.
